scripts/commands/checkLinks.ts

```diff
@@ -19,7 +19,7 @@ import { hideBin } from "yargs/helpers";
 import { pathExists } from "../lib/fs";
 import { File } from "../lib/links/LinkChecker";
 import { FileBatch } from "../lib/links/FileBatch";
-import { QISKIT_MISSING_VERSION_MAPPING } from "../lib/qiskitMetapackage";
```

Comment:
This constant can be deleted now, right? It's still good to have the dedicated file though.